MSAL/test/app/MSALTestAppSettings.m

```diff
@@ -176,24 +176,22 @@ - (MSALAccount *)accountForAccountHomeIdentifier:(NSString *)accountIdentifier
 
 - (void)readFromDefaults
 {
+    s_currentProfileIdx = 0;
+    s_currentProfile = [s_profiles objectForKey:[s_profileTitles objectAtIndex:s_currentProfileIdx]];
```

Comment:
nit: are both s_profiles and s_profileTitles guaranteed to have at least one item? if not, it will crash.

Reply:
Yes the initialize method is already taking care of ensuring that through the default profile.

Reply:
Yes the initialize method is already taking care of ensuring that through the default profile.